opennms-base-assembly/src/main/filtered/etc/vacuumd-configuration.xml

```diff
@@ -416,7 +428,16 @@
            AND statustime &lt; now() - interval '24 hours'
       </statement>
     </action>
-    
+
+    <action name="clearPathOutages" >
+      <statement>
+        UPDATE alarms
+           SET severity = 2, lastautomationtime = ${_ts}
+         WHERE eventuei = 'uei.opennms.org/nodes/pathOutage'
```

Comment:
Should we not also factor in the time of the `nodeUp` alarm here?

I could imaging a scenario, where a node is flapping up and down, and existing `nodeUp` alarm clears a newly created `pathOutage` alarm.